package.json

```diff
@@ -35,13 +35,14 @@
   ],
   "typings": "dist/components/index",
   "dependencies": {
-    "emotion": "9.2.4",
+    "memoize-one": "3.1.1",
     "react": "16.4.1",
+    "react-virtualized": "9.19.1",
+    "emotion": "9.2.4",
```

Comment:
I think this is incorrectly ordered and will re-order once you run npm install.

Reply:
I'll check that.